target/linux/ramips/image/mt7621.mk

```diff
@@ -443,66 +443,65 @@ define Device/netgear_ex6150
 endef
 TARGET_DEVICES += netgear_ex6150
 
-define Device/netgear_r6220
+define Device/netgear_r6xxx
   MTK_SOC := mt7621
   BLOCKSIZE := 128k
   PAGESIZE := 2048
   KERNEL_SIZE := 4096k
-  IMAGE_SIZE := 28672k
   UBINIZE_OPTS := -E 5
-  SERCOMM_HWID := AYA
-  SERCOMM_HWVER := A001
-  SERCOMM_SWVER := 0x0086
   IMAGES += factory.img kernel.bin rootfs.bin
   IMAGE/factory.img := pad-extra 2048k | append-kernel | pad-to 6144k | append-ubi | \
-	pad-to $$$$(BLOCKSIZE) | sercom-footer | pad-to 128 | zip R6220.bin | sercom-seal
+	pad-to $$$$(BLOCKSIZE) | sercom-footer | pad-to 128 | zip $$$$(DEVICE_MODEL).bin | sercom-seal
   IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
   IMAGE/kernel.bin := append-kernel
   IMAGE/rootfs.bin := append-ubi | check-size $$$$(IMAGE_SIZE)
```

Comment:
I would drop these two completely, as we now have working factory images.

Reply:
I would prefer to not mix this with this commit, but keep it cosmetical. It can be done easily afterwards.